docs/output.md

```diff
@@ -15,8 +15,9 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
     - [FastQC](#fastqc) - Read quality control
     - [Cutadapt](#cutadapt) - Primer trimming
     - [MultiQC](#multiqc) - Aggregate report describing results
-    - [DADA2](#dada2) - Infer Amplicon Sequence Variants (ASVs) and taxonomic classification
-      - [Barrnap](#barrnap) - Predict ribosomal RNA sequences
+    - [Amplicon sequence variant inferrence with DADA2](#Amplicon-sequence-variant-inferrence-with-DADA2) - Infer Amplicon Sequence Variants (ASVs) and taxonomic classification
+      - [Barrnap](#barrnap) - Predict ribosomal RNA sequences and optional filtering
+      - [Length filter](#length-filter) - Optionally, ASV can be filtered by length thresholds
```

Comment:
Is the indentation off here?

Reply:
Hm, indentation of `Length filter` is at the same level as `Barrnap` above & `ITSx` below. And those are sub-sections of `Amplicon sequence variant inferrence with DADA2`, that is not correct.
Maybe if I change the header `Amplicon sequence variant inferrence with DADA2` to `Amplicon sequence variant inferrence with DADA2 and subsequent filtering` than having the sub-sections `Barrnap` & `Length filter` & `ITSx` are fine, but the sub-section `assignSH` still doesn't fit.

Reply:
I attempted to fix this below.